flask_admin/contrib/fileadmin/azure.py

```diff
@@ -47,28 +53,29 @@ def __init__(self, container_name, connection_string):
         :param connection_string:
             Azure Blob Storage Connection String
         """
-
-        if not BlockBlobService:
-            raise ValueError(
-                "Could not import `azure.storage.blob`. "
-                "Enable `azure-blob-storage` integration "
-                "by installing `flask-admin[azure-blob-storage]`"
-            )
-
         self._container_name = container_name
         self._connection_string = connection_string
```

Comment:
One of the changes I made on the s3 admin side when bringing it up to date was to have `__init__` take a client instance rather than parameters that get passed the client.

Do you think we should do something similar here and accept an instance of `BlobServiceClient`, or is it still fine to just use the connection string?

Reply:
It would be, but this is scheduled to go out for the v2 release where we're making a bunch of breaking changes, so I'm ok with it.

If you'd be happy to, feel free :) 🙏